### PR TITLE
Migrate Link Google Account Tests

### PR DESCRIPTION
**Pull request by atharvar28**
Link to JIRA ticket if there is one:

### CODE REVIEW

File: gen3-integration-tests/services/fence.py [hunk 1]
```diff
@@ -345,3 +352,93 @@ def get_client_id_secret(self, client_name):
...
+        """Links google account with user account with expiration if provided"""
+        auth = Gen3Auth(refresh_token=pytest.api_keys[user], endpoint=self.BASE_URL)
+        headers = {
+            "Content-Type": "application/json",
+        }
+        if expires_in is not None:
+            url = f"{self.GOOGLE_LINK_REDIRECT}&expires_in={expires_in}"
+        else:
+            url = self.GOOGLE_LINK_REDIRECT
+        logger.info(f"URL: {url}")
```
Comment (krishnaa05) [inline gen3-integration-tests/services/fence.py:366]: Do we want to print the Google link? Would it be a security concern?

File: gen3-integration-tests/tests/test_link_google_account.py [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+from services.fence import Fence
+
+
+@pytest.mark.fence
+class TestLinkGoogleAccount:
+    fence = Fence()
+
+    def test_link_unlink_google_account(self):
+        """
+        Scenario:
```
Comment (krishnaa05) [inline gen3-integration-tests/tests/test_link_google_account.py:16]: Scenario names are missing for test cases :)

File: gen3-integration-tests/tests/test_link_google_account.py [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+    def test_unlink_unlinked_account(self):
+        """
+        Scenario:
+        Steps:
+            1. Unlink the account which does not have any linked account
+            2. Expect 404 status_code
+        """
+        status_code = self.fence.unlink_google_account(user="auxAcct2_account")
+        assert status_code == 404
```
Comment (krishnaa05) [inline gen3-integration-tests/tests/test_link_google_account.py:82]: can we add a error message for the assert

File: gen3-integration-tests/tests/test_link_google_account.py [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+    def test_extend_link_unlinked_account(self):
+        """
+        Scenario:
+        Steps:
+            1. Try to extend the expiration time of unlinked_account
+            2. Expect 404 status_code
+        """
+        status_code = self.fence.extend_expiration(user="main_account")
+        assert status_code == 404
```
Comment (krishnaa05) [inline gen3-integration-tests/tests/test_link_google_account.py:92]: can we add a error message for the assert

File: gen3-integration-tests/services/fence.py (hunk 1 above)
Comment (atharvar28) [reply gen3-integration-tests/services/fence.py:366]: No we dont want to print the google link! I will move it to debug in next commit

**Review by haraprasadj — APPROVED** [review]

**Pull request merged**